README.md

```diff
@@ -43,6 +43,143 @@ The documentation should help configure the BinderHub service to:
 [persistent binderhub chart]: https://github.com/gesiscss/persistent_binderhub
 [was added]: https://github.com/jupyterhub/binderhub/pull/666
 
+## Installation
+
+1. Add the `binderhub-service` chart repository to helm:
+
+   ```bash
+   helm repo add binderhub-service https://2i2c.org/binderhub-service
+   helm repo update
+   ```
+
+   Note this URL will change eventually, as binderhub-service is designed
+   to be a generic service, not something for use only by 2i2c.
+
+2. Install the latest development version of `binderhub-service` into a
+   namespace.
+
+   ```bash
+   helm upgrade \
+    --install \
+    --create-namespace \
+    --devel \
+    --wait \
+    --namespace <namespace>
+    <name> \
+    binderhub-service/binderhub-service
+   ```
+
+   This sets up a binderhub service, but not in a publicly visible way.
+
+3. Test that it's running by port-forwarding to the correct pod:
+
+   ```bash
+   kubectl -n <namespace> port-forward $(kubectl -n <namespace> get pod -l app.kubernetes.io/component=binderhub -o name) 8585:8585
+   ```
+
+   This should forward requests on port 8585 on your localhost, to the binder service running inside the pod. So if you go
+   to [localhost:8585](http://localhost:8585), you should see a binder styled page that says 404. If you do, _success!_.
+
+4. Create a docker repository for binderhub to push built images to. In this tutorial, we will be using Google Artifact Registry,
+   but binderhub supports using other registries.
+
+   Create a new Artifact Registry ([via this URL](https://console.cloud.google.com/artifacts/create-repo). Make sure you're in the correct project (look at the drop
+   down in the top bar). If this is the first time you are using Artifact Registry, it may ask you to enable the service.
+
+   In the repository creation page, give it a name (ideally same name you are using for
+   helm chart), select 'Docker' as the format, 'Standard' as the mode, 'Region'
```

Comment:
```suggestion
   dedicated to the chart installation), select 'Docker' as the format, 'Standard' as the mode, 'Region'
```

Like this, we avoid mixxing up the helm chart definition with the installation of it (a helm chart installation is referred to as a helm release in helm terminology, but its not so intuitive to use).